docs/samediff/templates/model-import.md

```diff
@@ -1 +1,62 @@
-# Getting started: importing TensorFlow and ONNX models into SameDiff
\ No newline at end of file
+---
+title: Getting started: importing TensorFlow and ONNX models into SameDiff
+short_title: Model import
+description: importing TensorFlow and ONNX models into SameDiff
+category: SameDiff
+weight: 3
+---
+
+# Getting started: importing TensorFlow models into SameDiff
+
+## What models can be imported into samediff
+
+Currently samediff supports the import of Tensorflow frozen graphs through the various Samediff.importFrozenTF methods. 
```

Comment:
Capitalization: `samediff` -> `SameDiff`
`Tensorflow` -> `TensorFlow`
`Samediff.importFrozenTF` -> `SameDiff.importFrozenTF`

Same things in a few other places in the doc, I won't flag others...